vcd/resource_vcd_tm_content_library.go

```diff
@@ -0,0 +1,269 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdTmContentLibrary() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcdTmContentLibraryCreate,
+		ReadContext:   resourceVcdTmContentLibraryRead,
+		UpdateContext: resourceVcdTmContentLibraryUpdate,
+		DeleteContext: resourceVcdTmContentLibraryDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcdTmContentLibraryImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true, // TODO: Update not supported
```

Comment:
There are many more missed updates, but only mentioning here.

```suggestion
				ForceNew:    true, // TODO: TM: Update not supported
```

Reply:
Fixed, thanks!